src/woo_publications/publications/admin.py

```diff
@@ -156,6 +158,60 @@ def reassess_retention_policy(
     )
 
 
+@admin.action(description=_("revoke the selected %(verbose_name_plural)s"))
+def revoke_instance(
```

Comment:
```suggestion
def revoke(
```
the `_instance` creates the impression that it only operates on a single instance